blaze/command/push.py

```diff
@@ -85,6 +95,36 @@ def test_push(args):
     return 0
 
 
+def push_preload_all_policy_generator() -> Callable[[EnvironmentConfig], Policy]:
```

Comment:
This method signature is not compatible with how the function is used (it expects to be passed a `List[PushGroup]` below). Should change the other generator to also accept an `EnvironmentConfig` and get its push groups from that, and make the changes in `test_push` and `_get_page_load_time_in_replay_server`

Reply:
made params of generator consistent. changing _get_results_in_replay_server, _test_push